main.cr

```diff
@@ -16,6 +16,7 @@ module Mare
       option "--print-perf", desc: "Print compiler performance info", type: Bool, default: false
       option "-o NAME", "--output=NAME", desc: "Name of the output binary"
       option "-p NAME", "--pass=NAME", desc: "Name of the compiler pass to target"
+      option "--target=TARGET", desc: "Target triple"
```

Comment:
Would probably be good to explicitly mention here that we don't yet support cross-compiling, because if I just saw this option present, I'd assume that I could cross-compile.

Reply:
I removed `--target` option again.